views/playlists/playlist.html

```
@@ -1,7 +1,7 @@
 <div id="contenido" class="animate__animated animate__fadeIn">
   <!-- Back Button and Title -->
   <div class="flex items-center gap-4 mb-8">
-    <button hx-get="/ui/playlists" hx-target="#contenido" hx-swap="innerHTML" hx-push-url="true"
+    <button hx-get="/playlists" hx-target="#contenido" hx-swap="innerHTML" hx-push-url="true"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)